metadata/index.html

```diff
@@ -1292,6 +1287,24 @@
               An <a>atomic property</a> that contains a single string that is the main datatype of the values of the cell. The value is a term defined in the default context representing a datatype URL. If the cell contains a list (ie <code>separator</code> is specified and not <code>null</code>) then this is the datatype of each value within the list. Conversion specifications MUST use the datatype of the value to determine the conversion of a cell into the target format. See <a href="#datatypes"></a> for more details.
             </p>
           </dd>
+          <dt id="cell-aboutUrl"><code>aboutUrl</code></dt>
+          <dd>
+            <p>A <a>URI template property</a> that MAY be used to create a unique identifier for each cell within a row when mapping data to other formats. See <a href="" class="sectionRef"></a> for details.</p>
+            <p class="note"><code>aboutUrl</code> is typically defined on a Schema or Table. If defined on individual Columns, care must be taken to ensure that transformed cell values maintain a semantic relationship.</p>
+          </dd>
+          <dt id="cell-propertyUrl"><code>propertyUrl</code></dt>
+          <dd>
+            <p>An <a>URI template property</a> that holds one or more strings that MAY be expanded to URIs for properties if the table is mapped to another format. The default value of <code>propertyUrl</code> is <code>"{#_name}"</code>.</p>
+            <p class="issue" data-number="96">
+              The model was changed from an <a>atomic property</a> a <a>URI template property</a>, which does not presently allow arrays of strings. Either <a>URI template property</a> needs to be updated to allow for an array of template strings, or <code>propertyUrl</code> needs to only allow a single value. We could also change this back to an <a>atomic property</a>.
+            </p>
+            <p class="note"><code>propertyUrl</code> is typically defined on a Column. If defined on a Schema, Table or Table Group, care must be taken to ensure that transformed cell values maintain an appropriate semantic relationship.</p>
+          </dd>
+          <dt id="cell-valueUrl"><code>valueUrl</code></dt>
+          <dd>
+            <p>An <a>URI template property</a> that is used to define one or more <cite><a href="http://www.w3.org/TR/rdf11-concepts/#dfn-resource">resources</a></cite> value of a cell instead of a literal values, as defined in [[!rdf-concepts]]. See <a href="#parsing-cells" class="sectionRef"></a>.</p>
```

Comment:
For non-RDF heads, I suggest this reads "...used to map the values of the cells into URLs. See ..." and then perhaps adding a note about the implications for mapping to RDF.